### ISSUE TITLE
node-gyp appears to be broken when executing 'npm install' through shellton

### ISSUE THREAD

**Issue by catdad** [catdad/shellton#20]
```javascript
var shellton = require('shellton');

shellton({
  task: 'npm install ref',
  stdout: process.stdout,
  stderr: process.stderr,
}, (err) => {
  console.log('done', err);
});
```

**Comment by catdad** [comment]
`node-gyp` can't be found on the path. This appears to only happen on node 8, and not on node 4... weird. Might be related to #21.

**Comment by catdad** [comment]
This seems to have to do with having `PATH`, `Path`, and `path` on the env... weird part is, all 3 are set to the same value, but setting only `PATH` makes it work fine. So I need to investigate a bit more.

**Comment by catdad** [comment]
Published in 4.0.0.
